src/components/TabletApp/LeftSide/UpcomingCards/UpcomingCards.tsx

```diff
@@ -34,6 +35,7 @@ const UpcomingCards = ({
 
     const [joiners, setJoiners] = useState<(string | undefined)[]>();
     const [surplus, setSurplus] = useState<number>();
+    const location = useLocation();
```

Comment:
location  is a bit vague for const name maybe?

Reply:
the constant name is ok. It is how on the official docs is used also

Reply:
This is fine as it is here